deploy/manifests/cert-manager.yaml

```diff
@@ -561,17 +560,16 @@ metadata:
   namespace: "cert-manager"
   labels:
     app: webhook
-    chart: webhook-v0.6.3
+    chart: webhook-v0.6.4
     release: cert-manager
-    heritage: Tiller
----
+    heritage: Tiller---
```

Comment:
This looks incorrect - added a comment to the chart above to explain why it's happened 😄